packages/dx-react-grid/src/components/group-panel-layout.jsx

```diff
@@ -107,7 +115,7 @@ export class GroupPanelLayout extends React.PureComponent {
             ref={element => element && this.itemRefs.push(element)}
             item={item}
             itemComponent={Item}
-            draggingEnabled={draggingEnabled}
+            draggingEnabled={draggingEnabled && columnGroupingEnabled(item.column.name)}
             onDragStart={() => this.onDragStart(item.column.name)}
```

Comment:
Maybe we should use `const itemColumnName = item.column.name` because this used three times?

Reply:
Makes sense)